hacks/count.js

```diff
@@ -1,3 +1,19 @@
+function delta(currentCount, previousCount) {
+    if (!mongo_hacker_config['count_deltas']) {
+        return "";
+    }
+    var delta = Number(currentCount - previousCount);
+    delta = isNaN(delta) ? colorize("(first count)", { color: 'blue' }) :
+        (delta == 0) ?  colorize("(=)", { color: 'blue' }) :
+            (delta > 0) ? colorize("(+" + delta.commify() + ")", { color: 'green' }) :
+                (delta < 0) ? colorize("(" + delta.commify() + ")", { color: 'red' }) :
+                    (delta + " not supported");
+    return " / " + delta.pad(21); // FIXME use "dynamic" padding
```

Comment:
I've done some prep work for doing the multi-column output/formatting in #149, as I didn't want to squeeze it all into this PR... if you could have a look and sign it off first, then I can continue with this issue... thanks!